auditbeat/module/auditd/audit_linux.go

```diff
@@ -147,14 +147,19 @@ func (ms *MetricSet) Run(reporter mb.PushReporterV2) {
 		return
 	}
 
-	out, statusC, err := ms.receiveEvents(reporter.Done())
+	out, err := ms.receiveEvents(reporter.Done())
 	if err != nil {
 		reporter.Error(err)
 		ms.log.Errorw("Failure receiving audit events", "error", err)
 		return
 	}
 
 	if ms.kernelLost.enabled {
+		client, err := libaudit.NewAuditClient(nil)
+		if err != nil {
+			reporter.Error(err)
+			ms.log.Errorw("Failure creating audit monitoring client", "error", err)
+		}
 		go func() {
 			timer := time.NewTicker(lostEventsUpdateInterval)
```

Comment:
The `client` should probably be closed when this goroutine exits.

Reply:
Done. Good catch